.github/workflows/e2e-test.yml

```diff
@@ -10,11 +10,9 @@ concurrency:
   cancel-in-progress: true
 
 jobs:
-  functional:
```

Comment:
Changing the job to first wait and then run the two e2e test steps in parallel. It saves a bit of resources..

https://github.com/actualbudget/actual/actions/runs/6041739141?pr=1622